### PR TITLE
Story5 build timer

### PR DESCRIPTION
**Pull request by ellie-r**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <script src="script.js" defer></script>
+    <link href="style.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Fredericka+the+Great&family=Creepster&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div id="button">Button</div>
```
Comment (jennymegan) [inline index.html:11]: is this just a "pretend" element to placehold for the homescreen?

File: style.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+#timer {
+    background-color: #BC5F04;
+    color: white;
```
Comment (jennymegan) [inline style.css]: hex color please

File: style.css [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    width: 160px;
+    font-size: 4rem;
+    text-align: center;
+    border-radius: 10px;
+    box-shadow: 3px 3px 7px 8px rgba(0,0,0,0.5);
+    /*font-family: "Fredericka The Great";*/
```
Comment (jennymegan) [inline style.css]: remove this if not needed

File: index.html (hunk 1 above)
Comment (ellie-r) [reply index.html:11]: yes, we'll change it to be the coffin when integrating to the game screen

**Review by jennymegan — APPROVED** [review]

**Pull request merged**